isyfact-standards-referenzimplementierung-doc/modules/ROOT/pages/security/Referenzimplementierung-Security.adoc

```diff
@@ -196,7 +191,7 @@ Mit den blau-gestrichelten Linien wird dargestellt, dass der Aufruf einer REST-A
 Methode über das Http-Protokoll erfolgt.
 
 [id="ref-impl-security-call-hierachy",reftext="{figure-caption} {counter:figures}"]
-image::ref-impl-security-call-hierarchy.svg[align="center"]
+image::security/ref-impl-security-call-hierarchy.svg[align="center"]
 
 Das Bild soll nicht nur zeigen, wie innerhalb einer jeden Anwendung die Aufruf-Hierarchie
 aussieht:
```

Comment:
Pro Satz nur eine Zeile
```suggestion
Das Bild soll nicht nur zeigen, wie innerhalb einer jeden Anwendung die Aufruf-Hierarchie aussieht:
```

Reply:
Das Komma muss in diesem Satz auch noch weg

Reply:
done